plugins/chrome-ux-report-backend/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@backstage/plugin-chrome-ux-report-backend",
+  "version": "0.1.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
```

Comment:
Always nice to have the extra metadata linking back to the repo from the package manager, e.g. https://github.com/backstage/backstage/blob/c2280095ccf71fc02585973905ea24e064b7c8bf/plugins/catalog-backend/package.json#L14